drivers/hud_spm_report/app/models/hud_spm_report/generators/fy2024/measure_one.rb

```diff
@@ -213,12 +213,15 @@ def run_question!
     end
 
     private def median(values)
```

Comment:
This implementation was incorrect:
```
> median([1,2,3])
=> 1 # should be 2
```

Reply:
Good catch.  I'm still unclear why ruby doesn't have median, mean methods for arrays.  This seems like something a low-level maths class or gem should have so people don't make stupid mistakes.